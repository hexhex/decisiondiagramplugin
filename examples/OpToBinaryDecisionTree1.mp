[common signature]
predicate: root/1;
predicate: innernode/1;
predicate: leafnode/2;
predicate: conditionaledge/5;
predicate: elseedge/2;

[belief base]
name: kb1;
mapping: "
	root(root).
	innernode(root).
	leafnode(leaf1, class1).
	leafnode(leaf2, class2).
	leafnode(leaf3, class3).
	conditionaledge(root, leaf1, x, \'<\', y).
	conditionaledge(root, leaf2, z, \'<\', y).
	elseedge(root, leaf3).
";

[merging plan]
{
	operator: tobinarydecisiontree;
	{
		kb1
	};
}
