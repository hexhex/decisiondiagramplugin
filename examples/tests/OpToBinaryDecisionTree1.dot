digraph {
     root_1 -> leaf3 [label="else"];
     root -> root_1 [label="else"];
     root_1 -> leaf2 [label="z<y"];
     root -> leaf1 [label="x<y"];
     leaf3 [label="leaf3 [class3]"];
     leaf2 [label="leaf2 [class2]"];
     leaf1 [label="leaf1 [class1]"];
     root_1 [label="root"];
     root [label="root"];
}
