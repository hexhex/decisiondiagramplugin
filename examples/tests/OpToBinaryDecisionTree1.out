{elseedge(root_1,leaf3), elseedge(root,root_1), conditionaledge(root_1,leaf2,z,"<",y), conditionaledge(root,leaf1,x,"<",y), leafnode(leaf3,class3), leafnode(leaf2,class2), leafnode(leaf1,class1), innernode(root_1), innernode(root), root(root)}
